src/docs/sdk/envelopes.mdx

```diff
@@ -378,10 +378,15 @@ file. It is always associated to an event or transaction.
 : *String, optional.* The content type of the attachment payload. Defaults to
   `application/octet-stream`.
 
-### Session
+### Session / Sessions
```

Comment:
Above, we have `Event` and `Transaction` also in separate headings -- I would opt to split it for consistency.